File: LeetCode-Programs/Minimum Distance to the Target Element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        ans=len(nums)
        for i in range(len(nums)):
            temp=0
            if nums[i]==target:
                temp=abs(i-start)
                if temp<ans:
                    ans=temp
        return ans

File: LeetCode-Programs/Minimum Distance to the Target Element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int ans=nums.size();
        for(int i=0; i<nums.size(); i++){
            int temp = 0;
            if (nums[i] == target){
                temp = abs(i-start);
                if (temp<ans)
                    ans=temp;
                }
            }
        return ans;
    }
};
